packages/shared/src/write-archive/index.ts

```diff
@@ -36,7 +36,19 @@ export async function writeTestResult(
   archive: ResourceArchive,
   chromaticStorybookParams: ChromaticStorybookParameters
 ) {
-  const { title, outputDir, pageUrl } = e2eTestInfo;
+  const { titlePath, outputDir, pageUrl } = e2eTestInfo;
+  let titleHasTruncated = false;
+  const titlePathWithoutFileExtensions: string[] = [];
+  titlePath.forEach((aTitle) => {
+    // we only want to remove stuff after spaces for the file name
+    if (aTitle.includes('.') && !titleHasTruncated) {
+      titlePathWithoutFileExtensions.push(aTitle.split('.')[0]);
+      titleHasTruncated = true;
+    } else {
+      titlePathWithoutFileExtensions.push(aTitle);
+    }
+  });
```

Comment:
This code assumes that directories don't have periods (`.`) in them. Technically directories can have periods, but I figured it was enough of an edge case to not worry about now. 

If that seems problematic, I can make this a little smarter so it only takes stuff after the period for the file name, but not for directories.